operator/charts/templates/_helpers.tpl

```diff
@@ -34,7 +34,15 @@ config.yaml: |
   {{- if .Values.config.authorizer.enabled }}
   authorizer:
     enabled: {{ .Values.config.authorizer.enabled }}
-    exemptServiceAccounts: {{ join "," .Values.config.authorizer.exemptServiceAccounts }}
+    reconcilerServiceAccountUserName: {{ printf "system:serviceaccount:%s:%s" .Release.Namespace .Values.serviceAccount.name }}
```

Comment:
just wondering what happens when via `AuthorizerConfig` a different reconcile service account name is specified. The chart no longer honors that value and instead uses something predefined. We have to do it only one way then. If this always going to be predefined then why have that as part of the `AuthorizerConfig`?